doc/cephfs/fs-nfs-exports.rst

```diff
@@ -17,15 +17,30 @@ Create NFS Ganesha Cluster
 
 .. code:: bash
 
-    $ ceph nfs cluster create <type=cephfs> <clusterid> [<placement>]
+    $ ceph nfs cluster create cephfs <clusterid> [<placement>]
```

Comment:
We intend to use the same interface for rgw. So don't remove the type.
```suggestion
    $ ceph nfs cluster create <type> <clusterid> [<placement>]
```

Reply:
done - please take another look